### PR TITLE
Update binutils on AT next

### PR DESCRIPTION
**Pull request by er-1**
Bump to version 2.31.51

Signed-off-by: Erwan Prioul <erwan@linux.ibm.com>

### CODE REVIEW

**Comment by er-1** [comment]
@tuliom done.

**Comment by er-1** [comment]
I squashed my commits into one.

File: configs/next/packages/binutils/sources [hunk 1]
```diff
@@ -20,8 +20,8 @@
 #
 
 ATSRC_PACKAGE_NAME="GNU Binutils"
-ATSRC_PACKAGE_VER=2.30.51
-ATSRC_PACKAGE_REV=c52441483779
+ATSRC_PACKAGE_VER=2.31.51
+ATSRC_PACKAGE_REV=674e55f7b11e
```
Comment (gftg85) [inline configs/next/packages/binutils/sources]: OK

```
$ git show 674e55f7b11e:bfd/version.m4
m4_define([BFD_VERSION], [2.31.51])
```

**Comment by tuliom** [comment]
I requested a new AT next RHEL7 ppc64le build.

**Pull request merged**